websites/G/Google Photos/presence.ts

```diff
@@ -0,0 +1,31 @@
+const presence = new Presence({
+    clientId: "925204937225416704"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingTimestamp
+    },
+    path = document.location.pathname;
+  if (path === "" || path === "/") presenceData.details = "Browsing photos";
+  else if (path.startsWith("/memory")) {
+    presenceData.details = "Playing back a memory:";
+    [presenceData.state] = document.title.split(/-/, 1);
+  } else if (path.indexOf("/photo") > -1)
+    presenceData.details = "Viewing a photo";
+  else if (path.startsWith("/search")) {
+    presenceData.details = "Searching for:";
+    [presenceData.state] = document.title.split(/-/, 1);
+  } else if (path === "/albums") presenceData.details = "Browsing albums";
+  else if (path.startsWith("/album")) {
+    presenceData.details = "Viewing an album:";
+    presenceData.state = (await presence.getSetting<boolean>("albumname"))
+      ? document.title.split(/-/, 1)[0]
+      : "(Hidden in presence settings)";
```

Comment:
Just use an if and don't set state if the setting is false

Reply:
done